components/notifications-service/habitat/plan.sh

```diff
@@ -23,16 +23,17 @@ pkg_build_deps=(
   # We have seen failures with notification service http request
   # with external services. This is because the erlang version was bumped to
   # v23.2. Hence pinning the version till we have a fix.
-  core/erlang/21.3/20200825051314
+  core/erlang
```

Comment:
We cannot move to the latest version of Erlang since this fails connecting to external services. 
If we want to use the latest version of Erlang we need to find the solution 
The error seems to be because the HTTPoison is not supporting sslv3/
Check the solution here:
https://elixirforum.com/t/erlang-v-23-0-and-httpoison-error/31615/7